src/metabase/util/product_feedback.clj

```diff
@@ -0,0 +1,52 @@
+(ns metabase.util.product-feedback
```

Comment:
I based this off `fetch-token-and-parse-body`, I'm not entirely sure what some things I copied do so any feedback is welcome